src/Jss.js

```diff
@@ -15,6 +15,11 @@ export default class Jss {
     this.sheets = new SheetsRegistry()
     this.plugins = new PluginsRegistry()
     this.uid = uid
+    if (__VERSION__) {
```

Comment:
I am sure we can do the same but in a non-ugly way, for e.g. by using .babelrc if its possible to do from there, otherwise by creating a script in bin/babel and doing the same by requiring babels api.